examples/tfengine/generated/folder_foundation/audit/variables.tf

```diff
@@ -12,10 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 variable "folder" {
   type        = string
-  description = "ID of the folder to apply the configuration."
+  description = <<EOF
+ID of parent GCP resource to apply the policy.
+Can be one of the organization ID or folder ID according to parent_type.
```

Comment:
It will be confusing as the variable is named `folder`.

Maybe change the schema description to something more generic like

"ID of the parent GCP resource to apply the configuration."

(Note should be configuration instead of policy at the end)

Reply:
Maybe later we can explore some sort of interpolation or something like that to replace `parent` with `folder` or `organization`. I'm just throwing a moonshot.

For now, I left it as you requested. Resolving.